docs/reference/search-application/apis/index.asciidoc

```diff
@@ -0,0 +1,9 @@
+[[search-application-apis]]
+== Search Application APIs
```

Comment:
```suggestion
= Search Application APIs
```

Per Slack discussion consider using level 1 heading here.

Reply:
@kderusso I suspect this wasn't a quick fix. I imagine there are some more complex heading-level issues, and I'm wondering if these files need to be included in a _parent_ index file?

I'd recommend building locally to debug properly. :)

Reply:
Maybe it's just a problem with the auto-preview functionality and that heading level change was unnecessary? Sorry can't be more helpful about this right now...